python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -1472,6 +1471,9 @@ def submit(self, keep_going=None, keep_on_fail=None, prepare=False):
             # split into subjobs
             rjobs = self._doSplitting()
 
+            if not rjobs or len(rjobs) == 1 and rjobs[0] is self:
```

Comment:
Why do we care if there is only one subjob? Is it ever the case that we have a splitter which simply sets itself as `rjobs[0]`?